### ISSUE TITLE
Update drivers to use new DSL

### ISSUE THREAD

**Issue by dennwc** [bblfsh/sdk#243]
Update drivers to use new transformation DSL and new ToNode helpers.

**TODO:**
- [ ] Go
- [ ] Java
- [ ] JavaScript
- [ ] PHP
- [ ] Python
- [ ] Ruby

**Comment by juanjux** [comment]
Bash driver too.

**Comment by dennwc** [comment]
Yeah, sorry, was thinking about UAST2 (that will not be done for Bash).

**Comment by juanjux** [comment]
Why not? Is one of the stable drivers.

**Comment by dennwc** [comment]
Sure, but we have no u2 tests for it yet, and the model of Bash execution is comparable to C++ preprocessor.

I would say that it might make sense to limit a set of drivers for now, and add it once we port these 6 major drivers.

**Comment by juanjux** [comment]
Ok, but we should migrate the bash driver ASAP after the migration is completed because, as said, it's considered part of the `--recommended` set.

**Comment by dennwc** [comment]
This is one of the reasons for https://github.com/bblfsh/sdk/issues/235, since in this case bumping SDK version would remove incompatible drivers from `--recommended` automatically (for a particular version of `bblfshd`).

**Comment by juanjux** [comment]
Yes, I don't mean it in the sense that it would be installed but in the sense that it was requested to be updated and complete recently.
